index.html

```diff
@@ -1820,24 +1820,24 @@ <h3>Lifecycle Details</h3>
 <a>verifiable credentials</a> from <a>holders</a> for its supported actions.
           </li>
           <li>
-The <a>issuer</a> issues a <a>verifiable credential</a> to the <a>holder</a>.
+The <a>issuer</a> issues some <a>verifiable credential(s)</a> to the <a>holder</a>.
           </li>
           <li>
-The <a>holder</a> might pass on its <a>verifiable credentials</a> to another
+The <a>holder</a> might pass on its <a>verifiable credential(s)</a> to another
 <a>holder</a>.
           </li>
           <li>
-The final <a>holder</a> presents its <a>verifiable credentials</a> to the
+The <a>holder</a> presents its <a>verifiable credential(s)</a> to the
```

Comment:
A <a>holder</a> presents one or more of its <a>verifiable credentials</a> to the

Note. Now that we have more than one holder we can no longer refer to "the" holder as it is ambiguous. Final holder was not ambiguous.

Reply:
"Final holder" was not ambiguous, but it implied a termination of holder-to-holder transfers was required for a holder to present a VC to a verifier, which requirement does not exist.  As I noted in my earlier comment, "The" seems not to be appropriate for any of these bullets.